firmware/examples/PWM/source/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include "L0_LowLevel/delay.hpp"
+#include "L1_Drivers/gpio.hpp"
+#include "L1_Drivers/pwm.hpp"
+#include "L2_Utilities/debug_print.hpp"
+
+int main(void)
+{
+    DEBUG_PRINT("Pwm Application Starting...");
+
+    DEBUG_PRINT("Creating Gpio powered LED...");
+    DEBUG_PRINT("This Gpio powered LED represents the maximum brightness");
+    DEBUG_PRINT("and is used as reference against the Pwm powered LED");
+    DEBUG_PRINT("during its sweep.");
+    // Connect a 470 ohm resistor between the power line and the anode
+    // of the LED. The cathode is then connected to the gpio and pwm pin.
```

Comment:
Rather than comment this out, you should print this to the user so they know what to do.